### PR TITLE
Replace links from the WordPress News to the ClassicPress Blog

### PR DESCRIPTION
**Pull request by dikiyforester**
## Description
Replaces links from the WordPress News to the ClassicPress Blog in the News widget on the Dashboard.

Changes number of fetching items from 1 to 5.

## Motivation and context
This is an early step of implementation #40. A small change to make news links consistent.
According to comment https://github.com/ClassicPress/ClassicPress/issues/40#issuecomment-441152556

## How has this been tested?
Needs to clear the transient cache before testing.
Or alternatively, comment out a block of code:
https://github.com/ClassicPress/ClassicPress/blob/3d681412745ea3c51c934d39458b17182b450316/src/wp-admin/includes/dashboard.php#L985-L988

## Types of changes
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: src/wp-admin/includes/dashboard.php [hunk 1]
```diff
@@ -1113,8 +1113,7 @@ function wp_dashboard_events_news() {
 		<?php
 			printf(
 				'<a href="%1$s" target="_blank">%2$s <span class="screen-reader-text">%3$s</span><span aria-hidden="true" class="dashicons dashicons-external"></span></a>',
-				/* translators: If a Rosetta site exists (e.g. https://es.wordpress.org/news/), then use that. Otherwise, leave untranslated. */
-				esc_url( _x( 'https://wordpress.org/news/', 'Events and News dashboard widget' ) ),
+				esc_url( _x( 'https://www.classicpress.net/blog/', 'Events and News dashboard widget' ) ),
```
Comment (nylen) [inline src/wp-admin/includes/dashboard.php]: Don't translate site URL (1 of 3)

File: src/wp-admin/includes/dashboard.php [hunk 2]
```diff
@@ -1285,7 +1284,7 @@ function wp_dashboard_primary() {
 			 *
 			 * @param string $link The widget's primary link URL.
 			 */
-			'link' => apply_filters( 'dashboard_primary_link', __( 'https://wordpress.org/news/' ) ),
+			'link' => apply_filters( 'dashboard_primary_link', __( 'https://www.classicpress.net/blog/' ) ),
```
Comment (nylen) [inline src/wp-admin/includes/dashboard.php]: Don't translate site URL (2 of 3)

File: src/wp-admin/includes/dashboard.php [hunk 3]
```diff
@@ -1294,7 +1293,7 @@ function wp_dashboard_primary() {
 			 *
 			 * @param string $url The widget's primary feed URL.
 			 */
-			'url' => apply_filters( 'dashboard_primary_feed', __( 'http://wordpress.org/news/feed/' ) ),
+			'url' => apply_filters( 'dashboard_primary_feed', __( 'https://www.classicpress.net/feed/' ) ),
```
Comment (nylen) [inline src/wp-admin/includes/dashboard.php]: Don't translate site URL (3 of 3)

**Comment by nylen** [comment]
Explanation of my comments above:  we shouldn't translate our site URLs, since they will be available in English only for the time being.

Previously:  https://github.com/ClassicPress/ClassicPress/issues/184#issuecomment-433973368

Apart from that, this PR looks good as a first step.  :+1:

**Comment by dikiyforester** [comment]
Thanks for reviewing, @nylen!

Gettext functions now removed from URLs.

**Comment by dikiyforester** [comment]
Thanks!

**Pull request merged**